src/views/canvas/CanvasPinnedBlock.js

```diff
@@ -17,10 +17,9 @@ const CanvasPinnedBlock = boneView.extend({
       this.g.zoomer, 
       "change:_alignmentScrollLeft change:_alignmentScrollTop change:alignmentWidth change:alignmentHeight", 
       function(model, value, options) {
-        if ((!(((typeof options !== "undefined" && options !== null) ? options.origin : undefined) != null)) || options.origin !== "canvasseq") {
-          return this.render();
-        }
-    });
+        return this.render();
+      }
+    );
```

Comment:
Just removed the guard checks here.
They do not work and is just here because of a copy-paste from an [older canvas' listener](https://github.com/pradeepnschrodinger/msa/blob/126c83139338b20a415f1c3334f28bda54b13f05/src/views/canvas/CanvasSeqBlock.js#L17).